konfera/models/event.py

```diff
@@ -53,6 +70,17 @@ def __str__(self):
     def duration(self):
         return (self.date_to - self.date_from).days + 1
 
+    @property
+    def cfp_open(self):
+        return self.cfp_allowed and self.cfp_end >= datetime.now(tz=timezone.utc)
+
+    def clean(self):
+        if self.cfp_allowed and self.cfp_end == datetime(2000, 1, 1, 0, 0, tzinfo=timezone.utc):
```

Comment:
If we change the model to be `blank=False, null=True` we can change the condition to `self.cfp_allowed and not self.cfp_end` which is more readable.